configs/config/benchmark/linear_image_classification/datasets/ucf101.yaml

```diff
@@ -0,0 +1,49 @@
+# @package _global_
```

Comment:
looks great! would it make sense to move these out to
`linear_image_classification/ucf101/eval_linear_ucf101.yaml` ? that will create more visibility to the task! we can reference any example paper that uses this benchmark. thoughts? : )

Reply:
I will try to get the same performance on SimCLR with UCF101 than reported inside the CLIP paper. If so, let's publish it, otherwise, the best is probably for this dataset to stay in the "datasets" folder only.